packages/Collapsible/README.md

```diff
@@ -0,0 +1,53 @@
+## Collapsible
+
+A controlled component that shows/hides content on click.
+
+### Installation
+
+`> npm install --save @paprika/Collapsible`
```

Comment:
this is an error you can't capitalize package on npm.

Reply:
Oops, I missed that too.  It's correct in the `package.json` so I don't think we need to worry too much. I'm going to take a pass at this too, when I use it for my `<ResponseAccordion>` component, so I can fix it then.